keyboards/handwired/dactyl_manuform/4x6_5/keymaps/default/keymap.c

```diff
@@ -0,0 +1,74 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

Reply:
It's a copy-paste from the other source code file in this repo. I can add my github handle here, but I'm not sure I'm the one to take the copyright for this. Or maybe I should do that anyway?

Reply:
I pushed the change with the copyright header in case it's the way to proceed. If it's not, I can change it back

Reply:
We want copyright headers for all source files (c, and h files).  There are files that dont, because they havent been updated yet.